test/Engine/ProtoTest/FFITests/CSFFITest.cs

```diff
@@ -1002,11 +1002,6 @@ public void DisposeOnFFITest006()
                             v = dv.GetValue();
                             ";
             thisTest.RunScriptSource(code);
-            var gcStrategy =  thisTest.SetupTestCore().Heap.GCStrategy;
```

Comment:
The diff here is incorrect, I didn't remove these code. Double-check and seems it is a diff problem.